prisma/migrations/20230226191138_add_redemption_kyc/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateEnum
+CREATE TYPE "KycStatus" AS ENUM ('NOT_STARTED', 'PASS', 'PENDING', 'FAIL_TRANSACTION', 'FAIL_MAX_ATTEMPTS');
+
+-- CreateEnum
+CREATE TYPE "DedupeStatus" AS ENUM ('NOT_STARTED', 'PASS', 'FAIL');
+
+-- CreateTable
+CREATE TABLE "redemptions" (
+    "id" SERIAL NOT NULL,
+    "created_at" TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "user_id" INTEGER NOT NULL,
+    "kyc_attempts" INTEGER NOT NULL DEFAULT 0,
+    "kyc_status" "KycStatus" NOT NULL,
+    "jumio_account_id" VARCHAR,
+    "public_address" VARCHAR NOT NULL,
+    "email_verified" BOOLEAN NOT NULL DEFAULT false,
+    "dedupe_status" "DedupeStatus" NOT NULL,
```

Comment:
Keep in mind a minority of users are going through this process. This avoids having to worry about filling in this field intelligently for the 100k+ users that we're essentially ignoring because they have no points or haven't verified email.

Reply:
There are additional heuristics I had in mind, and it felt weird to check that during redemption vs before a user requests anything (since I thought Mat was going to work on some manual checking). Open to discussing this further, though

Reply:
I think we should always check deduperun_status because it duplicates this. If we don't propagate this it when we may make a mistake and they could be out of sync.

Reply:
We decided to remove this since it already lives on eligibility_checks.status and we can just join to get it